package.json

```diff
@@ -30,6 +30,7 @@
   "devDependencies": {
     "@babel/core": "^7.5.4",
     "@babel/plugin-proposal-object-rest-spread": "^7.5.4",
+    "@babel/plugin-syntax-dynamic-import": "^7.8.3",
```

Comment:
While necessary, using the babel dynamic import plugin is not enough. You also need to use https://www.npmjs.com/package/@plotly/webpack-dash-dynamic-import and to add that plugin in your webpack config file like so https://github.com/plotly/dash-core-components/blob/dev/webpack.config.js#L134